k8s_handle/templating.py

```diff
@@ -79,17 +85,11 @@ def include_file(path):
     return env
 
 
-def _create_dir(dir):
```

Comment:
Not needed, one usage, hides bug with resource file create error due to PermissionError in case of change of TEMP_DIR.